### PR TITLE
Gatsby: list-card desktop styles

### PR DESCRIPTION
**Pull request by george-cz**
Desktop styles for list-card component as per design here:
https://www.figma.com/file/9avjIglqc1VVc84zHIoLha/CovidPortal?node-id=1045%3A12637

image
image

### CODE REVIEW

File: gatsby/src/components/list-card/measure-list-card.module.scss [hunk 1]
```diff
@@ -1,6 +1,44 @@
...
+@import '../../../assets/scss/design-system/vendor/bootstrap/variables';
+
 .measure-list-card-detail {
   font-size: 1.3rem;
   line-height: 2rem;
   font-weight: 500;
   color: #2362a2;
 }
+
+@include media-breakpoint-up(xl) {
```
Comment (rostaklein) [inline gatsby/src/components/list-card/measure-list-card.module.scss:13]: Not really sure about the breakpoint to be honest. Id probably go for something more narrow? Cause seeing mobile design at `1190px` feels weird :D Doesnt it?

Comment (vitkolos) [reply gatsby/src/components/list-card/measure-list-card.module.scss:13]: I totally agree with @rostaklein, it should be lg.

Comment (george-cz) [reply gatsby/src/components/list-card/measure-list-card.module.scss:13]: Gotcha, let me change it. xl felt the most natural, but we can always easily tweak this when we add the left menu :)

**Comment by rostaklein** [comment]
Could you please make a component out of the "ListCard"s? and attach a title to it? 🤔 
image

**Review by rostaklein — APPROVED** [review]
The layout is much better on desktop now :) Even though the hover effect could be a bit better i guess (no design for that though)

**Pull request merged**